### ISSUE TITLE
Different ranks with Aq skill books

### ISSUE THREAD

**Issue by GiggidyGamer** [wowsims/sod#1135]
I think it would be very handy to give control over which rank of spell you use as we move into the next phase with the skill books coming out. It would be very helpful to show the dps difference that each of the books give you. It will also let you get a realistic dps test for yourself if you are unlucky in aqcuiring the books but still want to see what the upper end of your dps could look like despite running a lower rank of heroic strike than max. Hope this can get added so I scratch the curiosity itch moving into aq. :^)

TLDR: Lemme choose which rank of heroic stike i use to change between rank 8 and 9 for aq, revenge too :)

**Comment by kayla-glick** [comment]
Hey @GiggidyGamer 👋 AQ ranks will be made available soon as we work on implementing AQ changes
